src/main/webapp/resources/js/pages/launch/components/ParametersModal/ParametersModal.jsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import isEqual from "lodash/isEqual";
+import { Form, Input, Modal, Space, Tag } from "antd";
+import { useLaunchDispatch, useLaunchState } from "../../launch-context";
+import { SPACE_SM } from "../../../../styles/spacing";
+import { ParametersFooter } from "./ParametersFooter";
+import { useResetFormOnCloseModal } from "../../../../hooks";
+
+/**
+ * React component to render a modal window for modifying and saving pipeline
+ * parameters.
+ *
+ * @param {boolean} visible - determines if the modal should be displayed or not.
+ * @param {function} closeModal - function to close the modal window.
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function ParametersModal({ visible, closeModal }) {
+  /*
+   The currently selected parameter set.  If it is the pipelines default set,
+   it can be used modified (but not saved over), or it can be saved as a new
+   parameter set.
+   */
+  const { parameterSet } = useLaunchState();
+  const {
+    dispatchUseSaveAs,
+    dispatchUseModifiedParameters,
+  } = useLaunchDispatch();
+
+  /*
+  Store a copy of the original values to compare against to see if they
+  have been modified.
+   */
+  const originalValues = parameterSet.parameters.reduce(
+    (a, b) => ({ ...a, [b.name]: b.value }),
+    {}
+  );
+
+  /*
+  Have any of the field been modified?
+   */
+  const [modified, setModified] = React.useState(false);
+
+  /*
+  Use by the ant form to set original values for the parameters.
```

Comment:
Typo. Changed to `Used`

Reply:
Fixed in 84c7e7860c649dce50072cc9191e39e3a46c522b